plugins/package-managers/node/src/main/kotlin/NodePackageManager.kt

```diff
@@ -25,6 +25,7 @@ import org.apache.logging.log4j.kotlin.logger
 
 import org.ossreviewtoolkit.analyzer.PackageManager
 import org.ossreviewtoolkit.analyzer.PackageManagerResult
+import org.ossreviewtoolkit.analyzer.determineEnabledPackageManagers
```

Comment:
commit-msg:

generall -> generally

Should the commit type be something else (not `fix`). It seems that the handling is new.

Reply:
Well, the new handling fixes the bug that previously NPM was always used as the fallback, even if that manager was disabled.